File: lua/calc_old.lua
local a = tonumber(io.read())
local b = io.read()
local c = tonumber(io.read())

if b == "+" then
  print(a + c)
elseif b == "-" then
  print(a - c)
elseif b == "*" then
  print(a * c)
elseif b == "/" then
  print(a / c)
else
  print("Your input is incorrect!")
end

File: ruby/calc_old.rb
a = gets.chomp.to_f
b = gets.chomp
c = gets.chomp.to_f

if b == "+"
  puts a + c
elsif b == "-"
  puts a - c
elsif b == "*"
  puts a * c
elsif b == "/"
  puts a / c
else
  puts "Your input is incorrect!"
end
